From a513ace04d65c36acf813688c1a424ee61c50b49 Mon Sep 17 00:00:00 2001
From: Mans Rullgard <mans@mansr.com>
Date: Tue, 29 Nov 2016 12:25:03 +0000
Subject: [PATCH 1/2] copyfile: do not close output file descriptor

The copyfile() function should not close the output file descriptor.
This is the responsibility of the caller which might need also need it
for further operations.

Signed-off-by: Mans Rullgard <mans@mansr.com>
---
 core/cpio_utils.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/cpio_utils.c b/core/cpio_utils.c
index 7a5064b992a7..e51a1f6e1658 100644
--- a/core/cpio_utils.c
+++ b/core/cpio_utils.c
@@ -278,8 +278,6 @@ copyfile_exit:
 	if (dgst)
 		swupdate_HASH_cleanup(dgst);
 
-	close(fdout);
-
 	return ret;
 }
 
-- 
2.10.2

